docs/get-started/setup.md

```diff
@@ -1,3 +1,116 @@
 ---
 title: 'Setup'
 ---
+
+Now that you’ve learned about what stories are and how to browse stories, let’s demo working on one of your components. Pick a simple component from your project, like a Button, and write a `.stories.js` file to go along with it. It might look something like this:
```

Comment:
```suggestion
Now that you’ve learned what stories are and how to browse them, let’s demo working on one of your components. Pick a simple component from your project, like a Button, and write a `.stories.js` file to go along with it. It might look something like this:
```

Reply:
Fixed with latest commit